app/src/main/res/layout/activity_detail_match.xml

```diff
@@ -234,6 +234,19 @@
         app:layout_constraintTop_toBottomOf="@+id/text_red"
         tools:text="69':Own  Jonjo Shelvey;61': Adam Lallana;51': Adam Lallana;33': Alberto Moreno;" />
 
```

Comment:
Replace hardcode value in **layout_height** and **layout_width** attirbute of `TextView` with `wrap_content`.